modules/rostests/apitests/gdi32/GetObject.c

```diff
@@ -344,6 +344,21 @@ Test_Brush(void)
 
     ok(GetObjectW((HANDLE)GDI_OBJECT_TYPE_BRUSH, sizeof(LOGBRUSH), &logbrush) == 0, "\n");
     ok(GetLastError() == ERROR_INVALID_PARAMETER, "expected ERROR_INVALID_PARAMETER, got %ld\n", GetLastError());
+
+    /* Test handle fixup */
+    hBrush = (HBRUSH)((ULONG_PTR)GetStockObject(WHITE_BRUSH) & 0xFFFF);
```

Comment:
Since it's exported, you could also test calling GdiFixUpHandle directly?

Reply:
I'm too lazy to write a new test